README.md

```diff
@@ -113,6 +113,17 @@ The HTTP methods in the below sample are supported.
 ```dart
   @GET("/tasks/{id}")
   Future<Task> getTask(@Path("id") String id);
+  
+  @GET('/demo')
+  Future<String> queries(@Queries() Map<String, dynamic> queries);
+
+  @GET("https://httpbin.org/get")
+  Future<String> namedExample(
+      @Query("apikey") String apiKey,
+      @Query("scope") String scope, 
+      @Query("type") String type,
+      @Query("from") int from}
```

Comment:
Please correct the syntax issue

Reply:
My bad, I'll fix it. However, the source branch has been deleted, I'll submit another PR with the correct syntax.

Reply:
#125 is the correct PR.